### PR TITLE
black formatter line length reduction

### PR DESCRIPTION
**Pull request by thundergolfer**
**motivation:**  80 chars fits on modal.com/docs code blocks. 120 does not. We could adjust the UI to be higher, but we're unlikely to go all the way to 120.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ pythonpath = [
 ]
 
 [tool.black]
-line-length = 120
+line-length = 80
```
Comment (erikbern) [inline pyproject.toml:12]: could probably remove this section – i assume this is the default anyway

**Comment by erikbern** [comment]
i generally kinda hate the default width of 80, but it actually makes sense for docs

LGTM!

File: pyproject.toml (hunk 1 above)
Comment (thundergolfer) [reply pyproject.toml:12]: >  Black defaults to 88 characters per line,

Comment (erikbern) [reply pyproject.toml:12]: that's so weird

**Pull request merged**